tests/javascript/test-main.js

```diff
@@ -0,0 +1,35 @@
+var allTestFiles = [];
+var TEST_REGEXP = /(spec|test)\.js$/i;
+
+// Get a list of all the test files to include
+Object.keys(window.__karma__.files).forEach(function (file) {
+    if (TEST_REGEXP.test(file)) {
+        // Normalize paths to RequireJS module names.
+        // If you require sub-dependencies of test files to be loaded as-is (requiring file extension)
+        // then do not normalize the paths
+        var normalizedTestModule = file.replace(/^\/base\/|\.js$/g, '');
+        allTestFiles.push(normalizedTestModule);
+    }
+});
+
+require.config({
+    // Karma serves files under /base, which is the basePath from your config file
+    baseUrl: '/base',
```

Comment:
Didn't change the baseUrl since one way or the other we will either have long ../../media/system like paths or the long ones that are already here like 'tests/javascript/node_modules/jquery/dist/jquery.min'.